fs/nfs/Kconfig

```diff
@@ -13,6 +13,15 @@ config NFS
 
 #if NFS
 
+config NFS_DONT_BIND_TCP_SOCKET
+	bool "Do not bind TCP socket"
```

Comment:
let's remove the prompt string to hide the option from defconfig

Reply:
Done.